demo/screens/Home.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import NavigationTestUtils from "react-navigation/NavigationTestUtils";
+import { shallow } from "enzyme";
+import Home from "./Home";
+
+describe("Home", () => {
+  jest.useFakeTimers();
+  beforeEach(() => {
+    NavigationTestUtils.resetInternalState();
```

Comment:
Yes, make sense.
Some questions:
1. Make sense use Screen suffix on file names to avoid name duplication (ex.: screen/ListLandsScreen.js and components/presentational/ListLands.js)?
2. Is to do that with all screen components?

Reply:
1. I like that suffix idea.
2. Yeah, I think we might as well do this with all screen components.

Thanks!

Reply:
Hey @pcowgill what do you think?

EthereumQuestionScreen
```
import React from "react";
import EthereumQuestion from "../components/presentational/EthereumQuestion";

export default class EthereumQuestionScreen extends React.Component {
  render() {
    return (
      <EthereumQuestion
        onSignUp={() => this.props.navigation.navigate("EthereumSignUp")}
        onSignIn={() => this.props.navigation.navigate("EthereumSignIn")}
      />
    );
  }
}
```

EthereumQuestion
```
import React from "react";
import { Button, StyleSheet, View } from "react-native";
import LargeText from "./LargeText";
import Colors from "../../constants/Colors";

export default function EthereumQuestion(props) {
  return (
    <View style={styles.container}>
      <LargeText>{`Are you new to Ethereum?`}</LargeText>
      <View style={styles.buttonView}>
        <Button title="Yep" onPress={props.onSignUp} />
      </View>
      <View style={styles.buttonView}>
        <Button title="Nope" onPress={props.onSignIn} />
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: Colors.backgroundColor,
    alignItems: "center",
    justifyContent: "center",
  },
  buttonView: {
    flexDirection: "row",
    marginTop: 20,
  },
});

```